components/style/core/base.less

```diff
@@ -61,6 +61,10 @@ a {
   cursor: pointer;
   transition: color .3s ease;
 
+  &:focus {
+    text-decoration: underline;
```

Comment:
FYI, you need to run `npm test -- -u` to update demo snapshot, that make ci successd.